client/src/elm/Pages/Prenatal/ProgressReport/View.elm

```diff
@@ -2128,5 +2128,21 @@ viewTreatementForDiagnosis language date measurements allDiagnoses diagnosis =
         DiagnosisSeverePreeclampsiaAfterRecheck ->
             referredToHospitalMessage
 
+        DiagnosisHyperemesisGravidumBySymptoms ->
+            -- @todo
```

Comment:
Note this if it's something you wanted to do here in this PR :)

Reply:
@nedSaf 
This is a placeholder for Progress report implementation.
Got a separate issue for it.